### ISSUE TITLE
TableViewCell.swift

### ISSUE THREAD

**Issue by chaz-clark** [ashfurrow/Collection-View-in-a-Table-View-Cell#39]
was the TableViewCell.swift file added as a new cocoa file to the project? i get lost in the tutorial as to how this was generated, as its not native when a table view gets generated.

**Comment by ashfurrow** [comment]
Yeah, I went to File -> New File and selected Swift file. Is that not working? Can you upload the project that's not working?

**Comment by johncpang** [comment]
`UITableViewController` does not provide `UICollectionViewDataSource` nor `UICollectionViewDelegate`.  You'll first set your custom view controller class, and add extension (`UICollectionViewDataSource` and `UICollectionViewDelegate`) to it.  Then you can setup the `data source` and `delegate` outlets from collection view to the view controller.
